feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -191,3 +192,28 @@ export enum CartValidationStatusCode {
   PRICING_ERROR = 'pricingError',
   UNRESOLVABLE_ISSUES = 'unresolvableIssues',
 }
+
+export interface AddToCartToastConfig {
+  timeout?: number;
+}
+
+export enum ADDED_TO_CART_FEEDBACK {
+  MODAL,
+  TOAST,
+  NONE,
+}
+export interface CartToastItem {
```

Comment:
If CartToastItem is only used with the AddToCartToastComponent and its service, perhaps we put `CartToastItem` in a model file next to these classes.  Lets discuss.